resources/lang/en/events/misc/wetlands.json

```
@@ -0,0 +1,44 @@
+[
+  {
+    "event_id": "wtlnd_misc_any_stingingnettleboop",
+    "location": ["wetlands"],
+    "season": ["newleaf", "greenleaf", "leaf-fall"],
+    "weight": 20,
+    "event_text": "Whilst m_c was poking around in the undergrowth, {PRONOUN/m_c/subject} ran nose first into a stinging nettle plant - Ouch!",
```

```suggestion
    "event_text": "Whilst poking around in the undergrowth, m_c ran nose-first into a stinging nettle plant - ouch!",
```
Again, implying injury but not tagging one?

---

Stinging nettle doesn't cause too severe of an injury, that's why I put it under misc since it's basically a minor bee sting, and if I were to put it under injuries we currently don't have an injury system that would work for this event with the stings! However I can temporarily move it into injuries in the meantime though